card/tool.py

```diff
@@ -12,7 +12,7 @@
 from .mount import Mount
 
 # Used for configuration & installation access
-PROTECTED_SSID = 'skynet'
+PROTECTED_SSID = "skynet"
```

Comment:
this should be "bushnet"!

would mind fixing please? (I know it's not part of your change)